src/OrchardCore/OrchardCore.Email.Abstractions/IEmailServiceEvents.cs

```diff
@@ -9,7 +9,6 @@ public interface IEmailServiceEvents
     /// </summary>
     /// <param name="message"></param>
     /// <param name="context"></param>
```

Comment:
These empty docs remain. These all can be removed.

Reply:
removed